src/Runner.Common/Terminal.cs

```diff
@@ -168,6 +170,10 @@ public void WriteSection(string message)
                 Console.WriteLine($"# {message}");
                 Console.WriteLine();
             }
+            else
```

Comment:
might be wired to change those errors get printed to STDOUT instead STDERR.

Reply:
> do we need to update all the Write*() functions in the file to have the else?
only the `Write*()` that don't have a `Trace.*` Instead of an `else`, should we actually Trace these always?


We could also do `Trace.Info($"WRITE SECTION # {message}");` and `Trace.Info($"WRITE SUCCESS √ {message}");` to keep it consistent with other traces in the other `Write*()` functions?

Reply:
I was more focused on the `WriteError()` parts. 😄